### DISCUSSION TITLE
how do I change the default tab length

### DISCUSSION THREAD

**Discussion by danie-dejager** [LazyVim/LazyVim#1415] [Q&A] [answered]
I'm new to neovim and LazyVim. I've looked at the documentation but could not see where I can configure the tab length. I prefer to use 2 or 4 spaces when I tab.

Please guide me how to configure how tab functions.

**Comment by dpetka2001** [comment, marked as answer]
Part of the Neovim [docs](https://neovim.io/doc/user/usr_30.html#30.5) regarding your issue. You should read from the docs to understand what you're doing. Another good one `:h 'tabstop'` will also give you plenty information.

**Reply by danie-dejager** [reply]
Thanks!
